src/braket/aws/aws_session.py

```diff
@@ -824,8 +824,18 @@ def copy_session(
                 region_name=new_region,
                 profile_name=profile_name,
             )
+        new_braket_client = (
+            boto_session.client(
+                "braket", region_name=new_region, endpoint_url=self.braket_client.meta._endpoint_url
```

Comment:
Can we do this without depending on the internal `_endpoint_url` of the boto client?

Reply:
The `endpoint_url` is only defined in Braket's boto client. If we don't want to depend `endpoint_url` on boto client, I think we could add `endpoint_url` as an attribute of AwsSession in addition to boto client.

Reply:
@shpface Any thought on this?